# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/base/SelectionManager.cpp
# Commit: dbdf17d42860
# Full Hash: dbdf17d42860765bcf2e6aaab1b0cd71ad1565b9
# Author: James Teh <jteh@mozilla.com>
# Date: 2025-02-07 16:40:09
# Regressor Bug: 1909142
# File Overlap Count: 1
# Description:
#   Bug 1909142 part 2: Expose custom highlights via accessibility APIs. r=morgan
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D233806
# ==============================================================================

diff -r 9529f52eebf3 -r dbdf17d42860 accessible/base/SelectionManager.cpp
--- a/accessible/base/SelectionManager.cpp	Fri Feb 07 05:25:15 2025 +0200
+++ b/accessible/base/SelectionManager.cpp	Fri Feb 07 02:03:56 2025 +0000
@@ -203,7 +203,8 @@
 bool SelectionManager::SelectionRangeChanged(SelectionType aType,
                                              const dom::AbstractRange& aRange) {
   if (aType != SelectionType::eSpellCheck &&
-      aType != SelectionType::eTargetText) {
+      aType != SelectionType::eTargetText &&
+      aType != SelectionType::eHighlight) {
     // We don't need to handle range changes for this selection type.
     return false;
   }